superduper/components/model.py

```diff
@@ -400,6 +400,13 @@ def __post_init__(self, db, example):
         if not self.identifier:
             raise Exception('_Predictor identifier must be non-empty')
 
+    def cleanup(self, db: "Datalayer") -> None:
+        """Clean up when the model is deleted.
+
+        :param db: Data layer instance to process.
+        """
+        db.cluster.compute.drop_component(self.uuid)
```

Comment:
I think we should call `db.cluster.drop_component` and all other related logic should be handled within `db.cluster.drop_component`.

Otherwise, we will sometimes call `cluster.drop_component` in some places, and at other times, we will call `cluster.compute.drop_component`.

WDYT?

Reply:
Yes, good point